tests/integration/helpers.py

```diff
@@ -257,20 +268,21 @@ async def deploy_cluster_components(ops_test: OpsTest) -> None:
     await ops_test.model.deploy(
         mongos_charm,
         num_units=0,
+        channel=channel,
         application_name=MONGOS_APP_NAME,
     )
     await ops_test.model.deploy(
         MONGODB_CHARM_NAME,
         application_name=CONFIG_SERVER_APP_NAME,
         channel="6/edge",
-        revision=173,
+        revision=192,
```

Comment:
I *think* it might be good to do *edge* instead, I feel conflicted though its up to you.

On one hand: if we use edge we can catch errors in the mongodb charm easier 
On the other: if we use edge we can have failing CI that blocks charm from being published

What do you think? I trust whichever option you think is best

Reply:
I think it's good to go to edge as well, I first upgraded because I had a bug but it was not related.
I'll try to go with edge :)